include/utils.php

```diff
@@ -5434,18 +5434,43 @@ function isValidId($id) {
 }
 
 function getAppString($key) {
-    
+
     global $app_strings;
-    
+
     if (!isset($app_strings[$key])) {
         LoggerManager::getLogger()->warn('Language key not found: ' . $key);
         return $key;
     }
-    
+
     if (!$app_strings[$key]) {
         LoggerManager::getLogger()->warn('Language string is empty at key: ' . $key);
         return $key;
     }
-    
+
     return $app_strings[$key];
-}
\ No newline at end of file
+}
+
+/**
+ * @param string $className
+ * @return mixed
+ */
+function getClassInstance($className, $includePath = false)
+{
+    global $log;
+
+    if ($includePath) {
+        require_once get_custom_file_if_exists($includePath);
+    }
+
+    $customClassName = 'Custom' . $className;
+
+    if (class_exists($customClassName)) {
+        return new $customClassName;
+    }
+
+    if (class_exists($className)) {
+        return new $className;
+    }
+
+    $log->fatal('Unable to find class: ' . $className);
```

Comment:
You can use LoggerManager::getLogger() method instead accessing to globals

Reply:
Thanks for pointing this out, fixed now.